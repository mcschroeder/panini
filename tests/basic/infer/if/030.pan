
f30 = let b = true in if b then 1 else 2

f31 = let b = false in if b then 1 else 2
