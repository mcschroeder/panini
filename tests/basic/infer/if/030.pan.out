f30 : {_:ℤ | _ = 1}
f31 : {_:ℤ | _ = 2}